src/elements/content-sidebar/MetadataInstanceEditor.tsx

```diff
@@ -0,0 +1,46 @@
+import {
+    MetadataInstanceForm,
+    type MetadataTemplateInstance,
+    UnsavedChangesModal,
+    type MetadataTemplate,
+} from '@box/metadata-editor';
+import React from 'react';
+
+export interface MetadataInstanceEditorProps {
+    isBoxAiSuggestionsEnabled: boolean;
+    isUnsavedChangesModalOpen: boolean;
+    template: MetadataTemplateInstance | MetadataTemplate;
```

Comment:
why it can be both? `MetadataTemplateInstance` seem suitable here 
Additionally, I think prop name should reflect that it's an instance as well

Reply:
Yes, but when you pick a template from `AddMetadataTemplateDropdown`, the one you choose has a `MetadataTemplate` type. So we decided to extend the types so the `MetadataInstanceEditor` can handle both - `MetadataTemplate` when creating a new instance, and `MetadataTemplateInstance` when editing an existing one.